src/include/ipc/probe.h

```diff
@@ -109,7 +110,9 @@ struct probe_dma {
  */
 struct probe_point {
 	uint32_t buffer_id;	/**< ID of buffer to which probe is attached */
-	uint32_t purpose;	/**< PROBE_PURPOSE_EXTRACTION or PROBE_PURPOSE_INJECTION */
+	uint32_t purpose;	/**< PROBE_PURPOSE_EXTRACTION, PROBE_PURPOSE_INJECTION or
+				  *  PROBE_PURPOSE_LOGGING
```

Comment:
Can you add TRACE here too (we can do in 1 ABI bump) as LOG and TRACE are different things in Zephyr.

Reply:
@lgirdwood I can add if it's ok to have it without matching implementation in this PR.